homeassistant/components/telegram_bot/strings.json

```diff
@@ -767,16 +870,17 @@
           "description": "Mark the message as a reply to a previous message.",
           "name": "Reply to message ID"
         },
-        "target": {
-          "description": "An array of pre-authorized chat IDs to send the notification to. If not present, first allowed chat ID is the default.",
-          "name": "Target"
-        },
         "title": {
           "description": "Optional title for your notification. Will be composed as '%title\\n%message'.",
           "name": "Title"
         }
       },
-      "name": "Send message"
+      "name": "Send message",
+      "sections": {
+        "custom_targets": {
+          "name": "Custom targets"
```

Comment:
What's custom targets?

Reply:
Ok, I read the PR description all the way to the end. I'd suggest to make a section called "advanced" instead to hide all advanced parameters.

Side note: I think more parameters belong under that section.